packages/wallet/shared/src/types/asset.ts

```diff
@@ -0,0 +1,19 @@
+export type Asset = {
```

Comment:
are these types used in backend as well?

Reply:
#1351 
 The Types shared here should be imported in the backend to be used as the response type of controllers, 
and also in the frontend in order to import and not duplicate response types.

So i believe these types should also be added to the controllers?